test/test_integration.py

```diff
@@ -219,9 +232,12 @@
                         }\n
                     }\n
                     """,
-                 'expect_out': 'col1,col2,col3\n1,2,3\n'}
+                 'expect_out': 'ref_id,full_reference\nRef_34,Emmo The University pp'
```

Comment:
If following recommended change above this would become something like `'col1,col2,col3\n"ab","cd","ef"'`.